File: voice-telephony/call-control/pause-resume-recording/code-samples/pauseResumeCallRecording.php
<?php
// https://developers.ringcentral.com/my-account.html#/applications
// Find your credentials at the above url, set them as environment variables, or enter them below

// PATH PARAMETERS
$accountId = '<ENTER VALUE>';
$telephonySessionId = '<ENTER VALUE>';
$partyId = '<ENTER VALUE>';
$recordingId = '<ENTER VALUE>';

// OPTIONAL QUERY PARAMETERS
$queryParams = array(
    //'brandId' => '~'
)

// POST BODY
$body = array(
    'active' => true
)

require('vendor/autoload.php');
$rcsdk = new RingCentral\SDK\SDK(getenv('clientId'), getenv('clientSecret'), getenv('serverURL'));
$platform = $rcsdk->platform();
$platform->login(getenv('username'), getenv('extension'), getenv('password'));
$r = $platform->patch("/restapi/v1.0/account/{$accountId}/telephony/sessions/{$telephonySessionId}/parties/{$partyId}/recordings/{$recordingId}", $body, $queryParams);
?>

File: voice-telephony/call-control/pause-resume-recording/code-samples/pauseResumeCallRecording.cs
// https://developers.ringcentral.com/my-account.html#/applications
// Find your credentials at the above url, set them as environment variables, or enter them below

// PATH PARAMETERS
string accountId = "<ENTER VALUE>";
string telephonySessionId = "<ENTER VALUE>";
string partyId = "<ENTER VALUE>";
string recordingId = "<ENTER VALUE>";

// OPTIONAL QUERY PARAMETERS
PauseResumeCallRecordingParameters pauseResumeCallRecordingParameters = new PauseResumeCallRecordingParameters {
    //brandId = '~'
};

// POST BODY
CallRecordingUpdate callRecordingUpdate = new CallRecordingUpdate {
    active = true
};

RestClient rc = new RestClient(
    Environment.GetEnvironmentVariable("clientId"),
    Environment.GetEnvironmentVariable("clientSecret"),
    false
);
await rc.Authorize(
    Environment.GetEnvironmentVariable("username"),
    Environment.GetEnvironmentVariable("extension"),
    Environment.GetEnvironmentVariable("password")
);
var r = await rc.Restapi().Account(accountId).Telephony().Sessions(telephonySessionId).Parties(partyId).Recordings(recordingId).Patch(callRecordingUpdate, pauseResumeCallRecordingParameters);
// PROCESS RESPONSE
